gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.8-bin.zip
```

Comment:
How about 8.10?

Reply:
I will work on it.

Reply:
[Gradle 8.10](https://docs.gradle.org/8.10/release-notes.html) needs java17, is it okay to upgrade?

Reply:
Thanks